server/src/DBService/LandingPage/Query.ts

```diff
@@ -3,7 +3,7 @@ import { gql } from "postgraphile";
 export const GET_USER_INVESTIGATIONS = gql`
 query InvestigationsInfoByUser($userName: String!) {
   userById(id: $userName) {
-    investigationsByLastUpdator {
+    investigationsByLastUpdator(filter: {investigationStatus: {notEqualTo: "טופלה"}}) {
```

Comment:
You can use the enum here

Reply:
There is no enum there

Reply:
I meant that you can use the investigation status enum there

Reply:
It isnt possible to insert ${} inside that kind of string. It can be sent in, but I think that isnt right to do so because that value is suppose to be a cons

Reply:
My bad, i got confused